.gitignore

```diff
@@ -3,3 +3,4 @@ demo/helper-scripts/*.pdf
 demo/helper-scripts/*.log
 /kustomization.yaml
 coverage.out
+.idea/
```

Comment:
I am using IntelliJ :)

Reply:
can you then make this a separate commit, since is not a change related to the PR description

Reply:
Added in https://github.com/kubernetes-sigs/node-feature-discovery/pull/1886